precompile/contract/interfaces.go

```diff
@@ -62,7 +62,7 @@ type AccessibleState interface {
 // ConfigurationBlockContext defines the interface required to configure a precompile.
 type ConfigurationBlockContext interface {
 	Number() *big.Int
-	Timestamp() uint64
+	Time() uint64
```

Comment:
I think this still introduces a breaking change (but slightly less). It still breaks precompiles that use Block.timestamp in their configuration. But I believe this is a far less problem and if it's required for libevm (not targeted for master), it should be fine.

Reply:
Look at the last commit: it uses core.NewBlockContext to go around breaking changes, such that this interface still has a Timestamp method and the Timestamp method is removed from the types.Block. It's pretty good I think?